File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--deploying-a-project-task-list-definition_1.cs
        public override void ItemAdded(SPItemEventProperties properties)
       {
           base.ItemAdded(properties);
           SPWeb web = properties.OpenWeb();
           properties.ListItem["Due Date"] = "July 1, 2009";
           properties.ListItem["Description"] = "This is a critical task.";
           properties.ListItem.Update(); 
       }  

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--deploying-a-project-task-list-definition_1.vb
    Public Overrides Sub ItemAdded(ByVal properties As SPItemEventProperties)
        MyBase.ItemAdded(properties)
        Dim web As SPWeb = properties.OpenWeb()
        properties.ListItem("Due Date") = "July 1, 2009"
        properties.ListItem("Description") = "This is a critical task."
        properties.ListItem.Update()
    End Sub